schemas/2014-04-01/Microsoft.Sql.json

```diff
@@ -0,0 +1,1388 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2014-04-01/Microsoft.Sql.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Sql",
+  "description": "Microsoft Sql Resource Types",
+  "resourceDefinitions": {
+    "servers": {
```

Comment:
>servers [](start = 5, length = 7)

Please add all the top level resource types to schemas/2015-01-01/deploymentTemplate.json and schemas/2014-04-01-preview.json.